### PR TITLE
Update kafo_parsers to 0.1.0

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
deb mirror of #1086

File: dependencies/jessie/kafo_parsers/control [hunk 1]
```diff
@@ -11,6 +11,7 @@ Package: ruby-kafo-parsers
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         puppet (<< 4.0.0)
+         ruby-json,
```
Comment (domcleal) [inline dependencies/jessie/kafo_parsers/control]: This shouldn't be needed with Ruby 1.9 or higher as JSON's built into Ruby, so all apart from precise can probably have it removed.

File: dependencies/jessie/kafo_parsers/control [hunk 2]
```diff
@@ -11,6 +11,7 @@ Package: ruby-kafo-parsers
...
+         ruby-rdoc
```
Comment (domcleal) [inline dependencies/jessie/kafo_parsers/control]: Probably the same reason this is missing, it's in Ruby 1.9.

**Comment by ares** [comment]
right, I removed rdoc and json deps from jessie control file

**Comment by domcleal** [comment]
Sorry, I hadn't meant to imply that it was only applicable to jessie.  The same comment should apply to wheezy and trusty too, since they're all 1.9 or higher platforms.  Only precise has a default Ruby version of 1.8.

**Comment by ares** [comment]
ok, fixed there as well, for some reason, they had rdoc before

**Comment by domcleal** [comment]
ruby-json looks correct now, it's now only on precise.

> ok, fixed there as well, for some reason, they had rdoc before

It looks like we ship a ruby-rdoc for precise, trusty and wheezy, so perhaps the version in the base Ruby package is too old.  A quick check of wheezy suggests the default version is 3.9.4 and 4.1.0 on jessie, while the gemspec requires >= 3.9.0 and the package requires >= 3.12.0.

So sorry, I think rdoc should probably stay unchanged from deb/develop - explicit dependencies on ruby-rdoc on precise, trusty and wheezy - but don't add it to jessie.  ruby-json should stay on precise only.

**Comment by ares** [comment]
I hope I have it right now :-)

**Comment by mmoll** [comment]
Works just fine for me on Debian/jessie, cherry-picked as 89dea28 into develop, building now. Díky @ares!

**Pull request closed without merging**